content/en/blog/_posts/2023-07-06-confidential-kubernetes.md

```diff
@@ -129,7 +129,7 @@ signature of the memory contents, which can be sent to the VM's owner as an atte
 the initial guest memory was not manipulated.
 
 The second generation of SEV, known as
-[Encrypted State](https://www.amd.com/system/files/TechDocs/Protecting%20VM%20Register%20State%20with%20SEV-ES.pdf)
+Encrypted State
```

Comment:
Instead of removing the link, we can also choose to include the PDF's new link as an alternative.
```suggestion
[Encrypted State](https://www.amd.com/content/dam/amd/en/documents/epyc-business-docs/white-papers/Protecting-VM-Register-State-with-SEV-ES.pdf)
```